docs/kyma/docs/036-inst-xip.io-domain.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Install Kyma on a GKE cluster with wildcard DNS
+type: Installation
+---
+
+If you want to try Kyma in a cluster environment without assigning the cluster to a domain you own, you can use [`xip.io`](www.xip.io) which provides a wildcard DNS for any IP address. Such scenario requires using a self-signed TLS certificate.
```

Comment:
The link doesn't load. Please change the address to http://xip.io/ . This comment applies to all the places where you link to xip.io.

Reply:
Good catch, thanks. Fixed.